tests/test_typing.py

```diff
@@ -66,3 +69,50 @@ def test_is_none_type():
     # `collections.abc.Callable` (even with python >= 3.9) as they behave
     # differently
     assert is_none_type(TypingCallable) is False
+
+
+class Hero:
+    pass
+
+
+class Team:
+    pass
+
+
+@pytest.mark.skipif(sys.version_info < (3, 9), reason='PEP585 generics only supported for python 3.9 and above.')
+def test_convert_generics():
+    assert convert_generics(int) == int
```

Comment:
can we parameterise these?

Reply:
We can but Python does actually run all of the stuff inside of `pytest.mark.parametrize` (even if the test should be skipped), so parametrized variables can only be strings, otherwise Python 3.7 and 3.8 tests will fail. 
Anyway, I've done the parametrization. It looks a little bit weird (but cleaner), involved `eval` magic, and also made me wrote `noqa: F401` to suppress `flake8`'s "unused import" warning. Maybe you have an idea for a better solution?